Assets/MRTK/Providers/XRSDK/Profiles/DefaultXRSDKInputSystemProfile.asset

```diff
@@ -20,9 +20,10 @@ MonoBehaviour:
     componentName: XRSDK Windows Mixed Reality Device Manager
     priority: 0
     runtimePlatform: 9
-    deviceManagerProfile: {fileID: 0}
+    deviceManagerProfile: {fileID: 11400000, guid: 8f48cb9c26fb518499c65c9f9e8bb4ed,
```

Comment:
Is this right? Looks like the WMR device manager is getting a profile.

Reply:
oh my mistake in checking this in I thought it was needed to fix a break with the oculus xrsdk device manager